dbms/src/Storages/Page/PageStorage.cpp

```diff
@@ -493,57 +496,104 @@ void PageWriter::writeIntoMixMode(WriteBatch && write_batch, WriteLimiterPtr wri
         }
         case WriteBatch::WriteType::REF:
         {
+            // 1. Check external page exist or not.
+            if (storage_v3->isPageIdExist(ns_id, write.ori_page_id, /*snapshot*/ nullptr))
```

Comment:
Why getNormalPageId can't get external page since there is related processing code here:
https://github.com/pingcap/tiflash/blob/bb572eba5d4cf4db000861b7ccfb70ac32df11a9/dbms/src/Storages/Page/V3/PageDirectory.cpp#L326

Reply:
If `check_prev` is false. Then after `del`,`getNormalPageId `  will get `del`.